servicecontrol/servicecontrol-in-practice.md

```diff
@@ -0,0 +1,27 @@
+title: ServiceControl in Practice
+summary: Details how to think about the use of ServiceControl in production environments
+tags:
+- ServiceControl
+---
+
+ServiceControl is a tool that can accommodate many situations. Certain features come with a cost and in some cases the cost of having a feature active is not worth the performance impact on the system.
```

Comment:
Trying to put a more positive spin on it :)

"ServiceControl is a tool that can accommodate many situations. There are many features that can be enabled or disabled depending on the needs of the environment. Each feature is valuable in certain contexts but also comes with a cost in terms of resource usage and performance. Each feature should be evaluated in each environment to determine if its value exceeds its cost."

The jump between situation-environment-context is a little clunky but you get the idea.